scripts/check_fork_node_wiki.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import sys
+
+from modularodm import Q
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.project.model import Node
+from website.addons.wiki.model import NodeWikiPage
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def get_fork_projects():
+    return Node.find(
```

Comment:
This would run _a lot_ faster if you used `framework.mongo.database` instead of modm

```python
return database.node.find({'is_fork': True, 'is_deleted': False}, {'wiki_page_versions: 1})
```